docs/ardana/testing.md

```diff
@@ -0,0 +1,32 @@
+# Ardana testing
+
+For SOC8, Ardana is deployed in a SUSEfied version (all source come from
+the internal build service).
+The current testing is far away from perfect and a lot of things are missing
+(eg pull request/changeset testing) but it's a first start to get a SUSEfied
+Ardana environment up and running.
+
+# Setup a custom enviroment
+We currently have a [Jenkins job](https://ci.suse.de/job/ardana-job/) (defined
+in the [automation git](https://github.com/SUSE-Cloud/automation/blob/master/jenkins/ci.suse.de/ardana-job.yaml)
+which can be used to create a new environment in the [engineering cloud](https://engcloud.prv.suse.net/).
+The deployment can then be accessed via the floating IP and used for debugging.
+
+**PLEASE DELETED YOUR ENVIRONMENT IN THE ENGCLOUD AFTER YOU ARE DONE**
+Otherwise we'll run into Quotas in our project.
```

Comment:
I would note that the job number ID is show in the stack list. Can be useful to detect the one that must be removed.